prebid/Prebid.js
# Sublime adapter debug

## Goal

Test `sublime.analytics` function existence and change version of adapter

## How to test

In Emilbus : 
- add this code line 20 before declaration of adUnits in `Emilbus/sac/resources/test-display/prebidjs/single.html` file : 
```javascript
var sublime = {};
```
and change sacHost: `sac.ayads.co.local` to sacHost: `sac.ayads.co`

In Prebid.js project
- launch `gulp build --modules=sublimeBidAdapter`
- copy paste `build/dist/prebid.js` content in `Emilbus/sac/resources/test-display/prebidjs/prebid.js`
- visits http://sac.ayads.co.local/resources/test-display/prebidjs/single.html?pbjs_debug=true
- verify in Network the call to http://sac.ayads.co.local/resources/test-display/prebidjs/prebid.js 
- you should have the version `0.3.1`

You shouldn't have any ads.

Remove code from `Emilbus/sac/resources/test-display/prebidjs/single.html` file :
```javascript
var sublime = {};
```
- visits http://sac.ayads.co.local/resources/test-display/prebidjs/single.html?pbjs_debug=true
- you should have an ad

## Critical barometer

- :ok_hand: code safe, none or little impact on live applications
